examples/bridge-app/linux/main.cpp

```diff
@@ -989,7 +989,8 @@ int main(int argc, char * argv[])
 
     // Setup composed device with two temperature sensors and a power source
     ComposedDevice ComposedDevice("Composed Device", "Bedroom");
-    DevicePowerSource ComposedPowerSource("Composed Power Source", "Bedroom", EMBER_AF_POWER_SOURCE_FEATURE_BATTERY);
+    DevicePowerSource ComposedPowerSource("Composed Power Source", "Bedroom",
+                                          to_underlying(PowerSource::PowerSourceFeature::kBattery));
```

Comment:
Ideally this would take a BitFlags of the right type (and return the `Raw()` from the `GetFeatureMap()` getter?).

Reply:
I have started to do that but when I `git grep GetFeatureMap` there are various examples that use a method called `GetFeatureMap` and expects a `uint32_t`
Most of those are *not* for this particular example (only one of them was) but that for consistency (and simplicity) I have kept this one as `uint32_t`.

In particular the accessor `{cluster_name}::Attributes::FeatureMap::Get` returns a `uint32_t` :(

Reply:
I meant specifically for the `DevicePowerSource` type.  But either way, that can be a separate PR.